samples/electron/app/window.js

```diff
@@ -23,7 +23,7 @@ function sendNotification(evt) {
       templateText: '<text>%s</text>',
       templateImage: '<image id="1" src="%s"/><text>This is a simple image toast notification example</text>'
     })
-	  strings = ['Hi from Electron', iconPath]
+	  strings = ['Hi from Electron', 'https://unsplash.it/64?image=883']
```

Comment:
This library has a bug - when I change to icon url to remote url, the image can't be loaded.